.babelrc

```diff
@@ -1,5 +1,17 @@
 {
     "env": {
+        "umd": {
```

Comment:
new env is not rly needed, just re-use `es`

Reply:
Alright, I just thought it's cleaner this way.

Reply:
i see `env` here as specifying needed babel transforms, not the target (output type) per se